server/webhook_notifier.go

```diff
@@ -109,6 +110,24 @@ func (w *WebhookNotifier) NotifyConnected(ctx context.Context, clientAddr net.Ad
 	return w.send(ctx, payload)
 }
 
+func (w *WebhookNotifier) NotifyDisconnected(ctx context.Context, clientAddr net.Addr, serverAddress string, playerInfo *PlayerInfo, backendHostPort string) error {
+	if w.requireUser && playerInfo == nil {
+		return nil
+	}
+
+	payload := &WebhookNotifierPayload{
+		Event:           WebhookEventConnecting,
```

Comment:
Actually, let's go with an event string of "disconnect", so declare a `WebhookEventDisconnecting` to pair up with the other.